ts/navigation/AuthenticatedStackNavigator.tsx

```diff
@@ -280,7 +280,8 @@ const AuthenticatedStackNavigator = () => {
         name={WalletDetailsRoutes.WALLET_DETAILS_MAIN}
         component={WalletDetailsNavigator}
         options={{
-          gestureEnabled: isGestureEnabled
+          gestureEnabled: isGestureEnabled,
+          headerShown: false
```

Comment:
```suggestion
       ...hideHeaderOptions   
```
I think it should be configured like this

Reply:
Addressed into ccca530ca08105f1340ae21e0425f3b139701c3a